classes/log/store.php

```diff
@@ -68,6 +78,17 @@ public function __construct(log_manager $manager) {
      *
      */
     protected function is_event_ignored(event_base $event) {
+
+        if ((!CLI_SCRIPT or PHPUNIT_TEST) and !$this->logguests) {
```

Comment:
No problem with that, but I copied from the database store which is Moodle core. Should I go on?